tutor/src/models/exercises/exercise.js

```diff
@@ -105,6 +105,10 @@ class TutorExercise extends BaseModel {
     return this.has_video;
   }
 
+  @computed get canCopyOrEdit() {
+    return !this.isMultiPart && this.isHomework;
+  }
+
```

Comment:
this also needs to check the `is_copyable` flag.  that will be true even if the user is the owner